gluon/tests/test_web.py

```diff
@@ -11,12 +11,12 @@
 import time
 import signal
 
-from fix_path import fix_sys_path
+from .fix_path import fix_sys_path
 
 fix_sys_path(__file__)
 
-from contrib.webclient import WebClient
-from urllib2 import HTTPError
+from gluon.contrib.webclient import WebClient
+from gluon._compat import urllib2
```

Comment:
Why keeping urllib2 in _compat and not use urllib? The objective is not to drop it once it is not required anymore (if ever possible once it need we drop support for 2.7 which may take time...)??